otter/controller.py

```diff
@@ -124,7 +124,19 @@ def pause_scaling_group(log, transaction_id, scaling_group, dispatcher):
                    conv_pause_group_eff(scaling_group, transaction_id))
 
 
-def resume_scaling_group(log, transaction_id, scaling_group):
+def conv_resume_group_eff(trans_id, group):
+    """
+    Resume scaling group of convergence enabled tenant
+    """
+    eff = parallel([
+        Effect(ModifyGroupStatePaused(group, False)),
+        mark_divergent(group.tenant_id, group.uuid).on(
```

Comment:
Yes. Sorry I struggled with this since `ConvergenceStarter` does that I didn't want to use it because it returns deferred. At the same time I didn't want to add it in `mark_divergent` since it `ConvergenceStarter` does it :confused:. So, I ended up putting it here. I guess it would be nice to put it in `mark_divergent` once all functions use it directly instead of `ConvergenceStarter`?